src/screens/points/content/PointsContent.tsx

```diff
@@ -476,15 +477,17 @@ const NextDistributionCountdown = ({ nextDistribution }: { nextDistribution: Dat
 
   return (
     <Text align="center" color="labelSecondary" size="17pt" weight="heavy">
-      {`${dayStr} ${hourStr} ${minuteStr}`}
+      {`${dayStr} ${hourStr} ${minuteStr}`.trim()}
```

Comment:
Where is the whitespace coming from?

Reply:
The `dayStr` is empty within 24h of the drop, and `hourStr` is empty within 1h of the drop, so the whitespace between d/h/m end up being included without `.trim()`

Reply:
makes sense!